src/kubernetes.cc

```diff
@@ -400,29 +389,30 @@ KubernetesReader::GetPodAndContainerMetadata(
   const json::Array* container_statuses =
       status->Get<json::Array>("containerStatuses");
 
-  // Move the container specs into a map from name to spec.
-  std::map<std::string, const json::Object*> container_spec_by_name;
-  for (const json::value& c_spec : *container_specs) {
-    const json::Object* container_spec = c_spec->As<json::Object>();
-    const std::string name = container_spec->Get<json::String>("name");
-    container_spec_by_name.emplace(name, container_spec);
+  // Move the container statuses into a map from name to status.
+  std::map<std::string, const json::Object*> container_status_by_name;
+  for (const json::value& c_status : *container_statuses) {
+    const json::Object* container_status = c_status->As<json::Object>();
+    const std::string name = container_status->Get<json::String>("name");
+    container_status_by_name.emplace(name, container_status);
   }
 
-  for (const json::value& c_status : *container_statuses) {
+  for (const json::value& c_spec : *container_specs) {
     if (config_.VerboseLogging()) {
-      LOG(INFO) << "Container: " << *c_status;
+      LOG(INFO) << "Container: " << *c_spec;
```

Comment:
Assuming the spec has a lot more in it than just the status -- might this make it too chatty?

Reply:
I agree with the comment, however, this is only being displayed if verbose logging is enabled, so I'm ok with keeping it.

Reply:
Heh, the status object isn't that small either. At some point the code logged both, so I figured I was just changing the order, but I guess the logging of the spec got lost.
Just pushed a commit that also logs the status. The config option says "**verbose** logging", so some chattiness should be expected, and it's helpful in debugging the processing of the incoming JSON.